lightning/src/ln/msgs.rs

```diff
@@ -454,11 +463,12 @@ impl NetAddress {
 			&NetAddress::IPv6 { .. } => { 18 },
 			&NetAddress::OnionV2(_) => { 12 },
 			&NetAddress::OnionV3 { .. } => { 37 },
+			&NetAddress::DNSHostname { .. } => { 258 }
```

Comment:
This should return the real serialized length, using the length of the hostname itself.